src/cascadia/TerminalApp/Tab.h

```diff
@@ -3,56 +3,65 @@
+            winrt::Microsoft::Terminal::TerminalControl::TermControl GetActiveTerminalControl() const;
+            std::optional<GUID> GetFocusedProfile() const noexcept;
 
-    bool IsFocused() const noexcept;
-    void SetFocused(const bool focused);
+            bool IsFocused() const noexcept;
+            void SetFocused(const bool focused);
 
-    winrt::fire_and_forget Scroll(const int delta);
+            winrt::fire_and_forget Scroll(const int delta);
 
-    bool CanSplitPane(winrt::TerminalApp::SplitState splitType);
-    void SplitPane(winrt::TerminalApp::SplitState splitType, const GUID& profile, winrt::Microsoft::Terminal::TerminalControl::TermControl& control);
+            bool CanSplitPane(winrt::TerminalApp::SplitState splitType);
+            void SplitPane(winrt::TerminalApp::SplitState splitType, const GUID& profile, winrt::Microsoft::Terminal::TerminalControl::TermControl& control);
 
-    winrt::fire_and_forget UpdateIcon(const winrt::hstring iconPath);
+            winrt::fire_and_forget UpdateIcon(const winrt::hstring iconPath);
 
-    float CalcSnappedDimension(const bool widthOrHeight, const float dimension) const;
+            float CalcSnappedDimension(const bool widthOrHeight, const float dimension) const;
 
-    void ResizeContent(const winrt::Windows::Foundation::Size& newSize);
-    void ResizePane(const winrt::TerminalApp::Direction& direction);
-    void NavigateFocus(const winrt::TerminalApp::Direction& direction);
+            void ResizeContent(const winrt::Windows::Foundation::Size& newSize);
+            void ResizePane(const winrt::TerminalApp::Direction& direction);
+            void NavigateFocus(const winrt::TerminalApp::Direction& direction);
 
-    void UpdateSettings(const winrt::Microsoft::Terminal::Settings::TerminalSettings& settings, const GUID& profile);
-    winrt::hstring GetActiveTitle() const;
-    winrt::fire_and_forget SetTabText(const winrt::hstring text);
+            void UpdateSettings(const winrt::Microsoft::Terminal::Settings::TerminalSettings& settings, const GUID& profile);
+            winrt::hstring GetActiveTitle() const;
+            winrt::fire_and_forget SetTabText(const winrt::hstring text);
 
-    void ClosePane();
+            void ClosePane();
 
-    WINRT_CALLBACK(Closed, winrt::Windows::Foundation::EventHandler<winrt::Windows::Foundation::IInspectable>);
-    DECLARE_EVENT(ActivePaneChanged, _ActivePaneChangedHandlers, winrt::delegate<>);
+            WINRT_CALLBACK(Closed, winrt::Windows::Foundation::EventHandler<winrt::Windows::Foundation::IInspectable>);
+            DECLARE_EVENT(ActivePaneChanged, _ActivePaneChangedHandlers, winrt::delegate<>);
 
-private:
-    std::shared_ptr<Pane> _rootPane{ nullptr };
-    std::shared_ptr<Pane> _activePane{ nullptr };
-    winrt::hstring _lastIconPath{};
+            DECLARE_EVENT(PropertyChanged, _PropertyChanged, Windows::UI::Xaml::Data::PropertyChangedEventHandler);

```

Comment:
You might be able to do both the `DECLARE_` and the `DEFINE_` for this particular event in one line with `WINRT_CALLBACK` (@DHowett-MSFT make sure I'm right about this)

Reply:
Good catch, `WINRT_CALLBACK` does seem to do the exact same thing, but in one macro.